scripts/test-renode.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+SCRIPT_SRC=$(realpath ${BASH_SOURCE[0]})
+SCRIPT_DIR=$(dirname $SCRIPT_SRC)
+TOP_DIR=$(realpath $SCRIPT_DIR/..)
+if [[ "${BASH_SOURCE[0]}" != "${0}" ]]; then
+	echo "You must run this script, rather then try to source it."
+	echo "$SCRIPT_SRC"
+	exit 1
+fi
+
+source $SCRIPT_DIR/build-common.sh
+
+init
+
+RENODE_BIN=${RENODE_BIN:-renode}
+RENODE_FOUND=false
+
+if [ -x "$RENODE_BIN" ]; then
+	RENODE_FOUND=true
+fi
+
+if command -v "$RENODE_BIN" 2>&1 1>/dev/null; then
+	RENODE_FOUND=true
+fi
+
+if ! $RENODE_FOUND; then
+	# Download prebuilt renode Release if none is currently installed
+	conda install -c antmicro -c conda-forge renode
+	RENODE_BIN=$CONDA_PREFIX/bin/renode
+fi
+
+case $CPU in
+	vexriscv | picorv32)
+		;;
+	*)
+        echo "Simulating the current configuration is not supported in Renode yet - skipping the test"
+        exit 0
+		;;
+esac
+
+LITEX_CONFIG_FILE="$TARGET_BUILD_DIR/test/csr.csv"
+if [ ! -f "$LITEX_CONFIG_FILE" ]; then
+	make firmware
+fi
+
+RENODE_SCRIPTS_DIR="$TOP_DIR/$TARGET_BUILD_DIR/renode"
+RENODE_RESC="$RENODE_SCRIPTS_DIR/litex_buildenv.resc"
+RENODE_REPL="$RENODE_SCRIPTS_DIR/litex_buildenv.repl"
+
+mkdir -p $RENODE_SCRIPTS_DIR
+python $TOP_DIR/third_party/litex-renode/generate-renode-scripts.py $LITEX_CONFIG_FILE \
+	--repl "$RENODE_REPL" \
+	--resc "$RENODE_RESC" \
+	--bios-binary "$TOP_DIR/$TARGET_BUILD_DIR/software/bios/bios.bin" \
+	--firmware-binary "$TOP_DIR/$TARGET_BUILD_DIR/software/$FIRMWARE/firmware.bin"
+
+# TODO: fix this - do not use hardcoded conda-based renode
+OPT_RENODE_LOCATION=$TOP_DIR/build/conda/opt/renode
+
+cd $OPT_RENODE_LOCATION
+
+# TODO: those files should be shipped together with Renode
```

Comment:
Can you add why these are needed?

Reply:
They are needed to run robot tests. The fact they are not in the conda package is a bug we need to fix.

wgetting the missing files is just a workaround to verify that everything else works as expected